### PR TITLE
add instr function

### PR DESCRIPTION
**Pull request by shaddockzy**
## What have you changed? (mandatory)
add function instr  in builtin_string/instr . I found it on #3275 

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
- Summarize your change (mandatory)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Separately describe each logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

Please **NOTE** that:
- Do not assume reviewers understand the original issue

## What are the type of the changes? (mandatory)
add func for tikv

The currently defined types are listed below, please pick one of the types for this PR by removing the others:
- New feature (change which adds functionality)
## How has this PR been tested? (mandatory)

Please describe the tests that you ran to verify your changes. Have you finished unit tests, integration tests, or manual tests? What additional tests would give you greater confidence in this change?
use manual tests

## Does this PR affect documentation (docs) update? (mandatory)
no

If there is document change, please file a PR in ([docs](https://github.com/tikv/tikv/tree/master/docs)) and add the PR number here.

## Does this PR affect tidb-ansible update? (mandatory)

If there is a configuration or metrics change, please file a PR in [tidb-ansible](https://github.com/pingcap/tidb-ansible), and add the PR number here.

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @shaddockzy 

Please sign off your all commits. See https://github.com/tikv/tikv/blob/master/CONTRIBUTING.md#signing-off-the-commit

If you have already pushed some commits before, you can use `git rebase -i master --signoff` to resign them off.

**Comment by siddontang** [comment]
please format your code to pass CI too.

File: src/coprocessor/dag/expr/builtin_string.rs [hunk 1]
```diff
@@ -3259,4 +3290,32 @@ mod tests {
...
+//            ("r", "foobArbar", 6),
+            //            ("", "foobArbar",0),
+            //            ("", "",0),
+            //            ("BaR", "foobArbar",0),
+            //            ("bar", "foobArbar",0),
+            //            (
+            //                "好世",
+            //                "你好世界",
+            //                0
+            //            ),
```
Comment (zhangjinpeng1987) [inline src/coprocessor/dag/expr/builtin_string.rs]: Why comment these lines?

**Comment by AndreMouche** [comment]
This PR will be closed since it's same with #4317

**Pull request closed without merging**